pkg/health/client/modules.go

```diff
@@ -6,12 +6,20 @@ package client
 import (
 	"fmt"
 	"io"
+	"regexp"
 	"strings"
 
+	"github.com/opentracing/opentracing-go/log"
```

Comment:
Any reason to use the opentracing log package for logging instead of `github.com/sirupsen/logrus` like most other cilium packages? AFAICT it's only used for the log line in `buildTree` below.

Reply:
Good catch! Thank you Tobias.